tool/README.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Publishing Devtool
+---
+
+* toc
+{:toc}
```

Comment:
We likely don't need the toc here - that will bring in the other pages, which are more related to using the tool, rather than publishing a new version.

Reply:
Good point, it's a README